host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_5.java

```diff
@@ -336,6 +336,7 @@ protected void readDomainElementAttributes(XMLExtendedStreamReader reader, Model
                         list.add(op);
                         break;
                     }
+                    case  ORGANIZATION: // not in the xsd but let's be forgiving to ease migration
```

Comment:
Not that I think that this will happen a lot, but if a customer did have both organization and domain-organization here, they'd just end up with whatever is defined last.